.gitignore

```diff
@@ -24,6 +24,8 @@
 /.ruby-gemset
 /exports/*
 !/exports/.keep
+.byebug_history
+/spec/examples.txt
```

Comment:
This file allows RSpec to persist some state between runs, it is configured in the spec_helper.rb. To be honest I'm tempted to disable it.